lib/tile_bag.rb

```diff
@@ -0,0 +1,40 @@
+module Scrabble
+  class TileBag
+    attr_reader :tile_bag
+
+    def initialize
+      @tile_bag = Scrabble::TileBag.create_tile_bag
+    end
+
+    def self.create_tile_bag
+      letters = ("a".."z").to_a
+      quantity= [9,2,2,4,12,2,3,2,9,1,1,4,2,6,8,2,1,6,4,6,4,2,2,1,2,1]
+      tile_bag = []
+
+      26.times do |i|
```

Comment:
Having that `26` sitting around makes me nervous - why not use `letters.length.times` instead?